source/_components/sensor.mold_indicator.markdown

```diff
@@ -21,6 +21,7 @@ The sensor data may be used e.g., to signal bad air quality (too high air humidi
 # Example configuration.yaml entry
 sensor:
   - platform: mold_indicator
+    name: Mould Sensor
```

Comment:
Please remove this line from the example, as per [the documentation](https://developers.home-assistant.io/docs/en/documentation_create_page.html#configuration):

> This sample must contain only the required variables to make it easy to copy and paste it for users into their configuration.yaml file.